app/views/pair_requests/_form.html.erb

```diff
@@ -12,20 +12,32 @@
         </ul>
       </div>
     <% end %>
-    <div id="form-container" class='flex flex-col space-y-4 xl:space-y-0 xl:max-w-full xl:inline-flex xl:justify-around xl:space-x-8 xl:flex-row'>
+    <div id="form-container" class='flex flex-col space-y-4 xl:space-y-0 xl:max-w-full xl:inline-flex xl:justify-around xl:space-x-8 xl:flex-row' data-controller="pair-requests-form">
       <div class="flex justify-between space-x-4 align-middle">
         <%= form.label :invitee_id, class: "label label-text" %>
         <%= form.select :invitee_id, 
                         User.invitee_select_for(current_user), 
```

Comment:
Just a thought (which could be entirely wrong), but would it be possible to include the timezone in the `invitee_select_for` call, and then store that as a custom data-* attribute so we don't need to make a seperate call to grab a specific user? Is this a valid javascript option here? (unsure if selects can have data-* options per choice).